storehaus-postgresql/src/main/scala/com/twitter/storehaus/postgres/PostgresValue.scala

```diff
@@ -0,0 +1,14 @@
+package com.twitter.storehaus.postgres
+
+import roc.postgresql.Element
+
+/**
+  * General trait to represent
+  * Postgres column value.
+  * NOTE: Actual implementation depends on driver for PostgreSQL
+  *
+  * Created by ponkin on 10/18/16.
+  */
+sealed trait PostgresValue
```

Comment:
I see. That's fine if we are going to automatically handle the conversions I think.